examples/astro/src/layouts/Layout.astro

```diff
@@ -14,6 +14,19 @@ const { title } = Astro.props as Props;
 		<link rel="icon" type="image/x-icon" href="/favicon.ico" />
 		<meta name="generator" content={Astro.generator} />
 		<title>{title}</title>
+		<script>
+			import { webVitals } from "../lib/vitals";
+
+			let analyticsId = import.meta.env.PUBLIC_VERCEL_ANALYTICS_ID;
```

Comment:
Have you tested this works? 🙏

Reply:
Yep! I actually ported it initially for the site I'm working on for work, then forked and added it to this repo. 

Screenshots:

![image](https://user-images.githubusercontent.com/19896267/187216473-21644c69-4808-43ab-83c4-d9c0a01fcc3c.png)

![image](https://user-images.githubusercontent.com/19896267/187216569-c026aed7-6bd2-49f7-b087-049cb62a374e.png)

Reply:
[Relevant section in Astro docs](https://docs.astro.build/en/core-concepts/astro-components/#client-side-scripts):

> By default, <script> tags are processed by Astro.
>
> Any imports will be bundled, allowing you to import local files or Node modules.
> -  The processed script will be injected into your page’s <head> with type="module".
> - TypeScript is fully supported, including importing TypeScript files
> - If your component is used several times on a page, the script tag will only be included once.
>
> ```html
> <script>
>   // Processed! Bundled! TypeScript-supported! ESM imports work, even to npm packages.
> </script>
> ```